components/dropdown/dropdown-content-mixin.js

```diff
@@ -678,20 +711,20 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 					// allow for outer margin
 					right: document.documentElement.clientWidth - targetRect.right - 15
 				}, spaceRequired, targetRect);
+
 				spaceAroundScroll = this._constrainSpaceAround({
 					above: targetRect.top + document.documentElement.scrollTop,
 					below: scrollHeight - targetRect.bottom - document.documentElement.scrollTop
 				}, spaceRequired, targetRect);
+
 			}
 
 			if (!ignoreVertical) {
 				this.openedAbove = this._getOpenedAbove(spaceAround, spaceAroundScroll, spaceRequired);
 			}
 
-			const position = this._getPosition(spaceAround, targetRect.width, contentRect.width);
-			if (position !== null) {
-				this._position = position;
-			}
+			this._position = this._getPosition(spaceAround, targetRect, contentRect);
```

Comment:
`_getPosition()` has been updated to return an object with the position properties. Previously it was just returning the relative `left` value, but for fixed positioning, it will provide `top`, etc.